src/main/java/org/randywebb/wlts/beans/DetailedMember.java

```diff
@@ -20,916 +20,773 @@
-		}
-		if (getStreet() != null) {
-			builder.append("street=");
-			builder.append(getStreet());
-			builder.append(", ");
-		}
-		if (getCity() != null) {
-			builder.append("city=");
-			builder.append(getCity());
-			builder.append(", ");
-		}
-		if (getState() != null) {
-			builder.append("state=");
-			builder.append(getState());
-			builder.append(", ");
-		}
-		if (getZip() != null) {
-			builder.append("zip=");
-			builder.append(getZip());
-			builder.append(", ");
-		}
-		if (getGivenName() != null) {
-			builder.append("givenName=");
-			builder.append(getGivenName());
-			builder.append(", ");
-		}
-		if (getCoupleName() != null) {
-			builder.append("coupleName=");
-			builder.append(getCoupleName());
-			builder.append(", ");
-		}
-		if (getHouseholdId() != null) {
-			builder.append("householdId=");
-			builder.append(getHouseholdId());
-			builder.append(", ");
-		}
-		if (getIsHead() != null) {
-			builder.append("isHead=");
-			builder.append(getIsHead());
-			builder.append(", ");
-		}
-		if (getIsSpouse() != null) {
-			builder.append("isSpouse=");
-			builder.append(getIsSpouse());
-			builder.append(", ");
-		}
-		if (getIsAdult() != null) {
-			builder.append("isAdult=");
-			builder.append(getIsAdult());
-			builder.append(", ");
-		}
-		if (getFullTimeMissionary() != null) {
-			builder.append("fullTimeMissionary=");
-			builder.append(getFullTimeMissionary());
-			builder.append(", ");
-		}
-		if (getFormattedMRN() != null) {
-			builder.append("formattedMRN=");
-			builder.append(getFormattedMRN());
-			builder.append(", ");
-		}
-		if (getSetApart() != null) {
-			builder.append("setApart=");
-			builder.append(getSetApart());
-			builder.append(", ");
-		}
-		if (getFormattedBirthDateFull() != null) {
-			builder.append("formattedBirthDateFull=");
-			builder.append(getFormattedBirthDateFull());
-			builder.append(", ");
-		}
-		if (getFormattedBirthDateFull() != null) {
-			builder.append("sustainedDate=");
-			builder.append(getFormattedBirthDateFull());
-		}
-		builder.append("]");
-		return builder.toString();
-	}
-
-	/**
-	* @return the street
-	*/
-	public String getStreet() {
-		return get("street");
-	}
-
-	/**
-	* @param street
-	*          the street to set
-	*/
-	public void setStreet(String street) {
-		put("street", street);
-	}
-
-	/**
-	* @return the city
-	*/
-	public String getCity() {
-		return get("city");
-	}
-
-	/**
-	* @param city
-	*          the city to set
-	*/
-	public void setCity(String city) {
-		put("city", city);
-	}
-
-	/**
-	* @return the state
-	*/
-	public String getState() {
-		return get("state");
-	}
-
-	/**
-	* @param state
-	*          the state to set
-	*/
-	public void setState(String state) {
-		put("state", state);
-	}
-
-	/**
-	* @return the zip
-	*/
-	public String getZip() {
-		return get("zip");
-	}
-
-	/**
-	* @param zip
-	*          the zip to set
-	*/
-	public void setZip(String zip) {
-		put("zip", zip);
-	}
+    /** For logging if needed during debugging. */
```

Comment:
This should be a comment, not a javadoc comment (// for a single-line comment or /*...*/ for a block comment

Reply:
Yep. I'll change that

Reply:
Cool, there's a bunch of these throughout the code. I didn't comment on all of them.
I guess since it is a class-level variable (private static Logger), one could argue that it should be a Javadoc comment, and I could buy that argument.
Usually, I consider Javadoc as API documentation and want it on all public entities. I don't typically require it for private entities, though it is OK to do it.